Packs/SecureWorks/Integrations/TaegisXDRv2/README.md

```diff
@@ -773,6 +776,9 @@ At least one of the inputs `alerts`, `events`, or `alert_query` MUST be defined
 | prioirity | The priority of the Investigation (1-5) | False |
 | status | The current status of the Investigation | False |
 | assigneeId | The id of a user to assign, in `auth0|12345` format | False |
+| serviceDeskId | A 3rd party ticket number for tracking purposes | False |
```

Comment:
Sorry for that, but before I can merge, I now noticed that your arguments don't follow our argument convention of underscore instead of camelCase please change the arguments in all needed places to follow this convention. 
service_desk_id

Reply:
@reedium.
Thanks for this clarification. 
I will go ahead and proceed with the merge process. 
Breaking backward compatibility is our last result.
I thought about asking you to change only the new arguments, but I rather keep the integration consistent with all the other commands and arguments.